cap/modules/records/serializers/schemas/json.py

```diff
@@ -27,8 +27,8 @@ class MetadataSchemaV1(Schema):
 
     def get_general_title(self, obj):
         """Get type title."""
-
-        collection = obj.get('collections', [])[0]
+        collection = obj.get('collections', [None])[0]
+        collection_mappings = None
```

Comment:
Since the next conditional will be always executed because of:
```python
>>> bool([None])                                                                                                                                             
True
```
You can just write this line like:
```python
collection_mappings = CAP_RECORD_MAPPINGS.get(collection, None)
```
And remove the next two lines ;)